### PR TITLE
Find cameras connected to computer after server start

### PR DESCRIPTION
**Pull request by bazile-clyde**

### CODE REVIEW

File: components/camera/videosource/webcam.go [hunk 1]
```diff
@@ -53,6 +54,7 @@ type CameraConfig struct {
 
 // Discover webcam attributes.
 func Discover(_ context.Context, getDrivers func() []driver.Driver, logger golog.Logger) (*pb.Webcams, error) {
+	mediadevicescamera.Initialize()
```
Comment (bazile-clyde) [inline components/camera/videosource/webcam.go:57]: This function initializes the re-discovery process.

File: components/camera/videosource/webcam.go [hunk 2]
```diff
@@ -386,13 +389,19 @@ func (c *monitoredWebcam) isCameraConnected() (bool, error) {
...
 	}
-	props, err := gostream.PropertiesFromMediaSource[image.Image, prop.Video](c.underlyingSource)
+	d, err := gostream.DriverFromMediaSource[image.Image, prop.Video](c.underlyingSource)
 	if err != nil {
-		return true, errors.Wrap(err, "cannot get properties from media source")
+		return true, errors.Wrap(err, "cannot get driver from media source")
 	}
-	// github.com/pion/mediadevices connects to the OS to get the props for a driver. On disconnect props will be empty.
+
 	// TODO(RSDK-1959): this only works for linux
```
Comment (bazile-clyde) [inline components/camera/videosource/webcam.go:397]: For Mac OS X this is still a no-op just as before. It should still trivially return as always connected.

File: components/camera/videosource/webcam.go [hunk 3]
```diff
@@ -386,13 +389,19 @@ func (c *monitoredWebcam) isCameraConnected() (bool, error) {
...
-	return len(props) != 0, nil
+	_, err = driver.IsAvailable(d)
+	switch {
+	case errors.Is(err, availability.ErrNoDevice):
```
Comment (edaniels) [inline components/camera/videosource/webcam.go:400]: given this is a boolean state, we can just if check the errors.Is

**Review by edaniels — APPROVED** [review]
LGTM except a thing

**Pull request merged**